.circleci/config.yml

```diff
@@ -373,61 +381,73 @@ workflows:
       - license-scan
       - build-container:
           name: "build-nsmd"
+          namespace: "nsm-system-integration"
```

Comment:
Nope. You don't need this set here for the build. You want it for the integration tests.